src/app/profile/yourGiving/givingRecipientView/donationRecipient/donationRecipient.tpl.html

```diff
@@ -0,0 +1,88 @@
+<div class="panel panel-margin-bottom recent-recipient-row">
+  <div class="panel-body">
+
+    <!-- Recipient Row -->
+    <div class="row">
+      <div class="col-md-2 col-full-height col-sm-12 col-xs-12 mb">
+        <img class="full-width" desig-src="{{$ctrl.recipient['designation-number']}}">
+      </div>
+      <div class="col-sm-12 col-md-7 col-full-height">
+        <h4 class="mb0">{{$ctrl.recipient['designation-name']}}</h4>
+        <small>#{{$ctrl.recipient['designation-number']}}</small>
+
+        <div class="row">
+          <div class="col-sm-3 col-md-3">
+            <div class="form-group mb0">
+              <label translate>Year-to-date:</label>
+              <ul class="list-unstyled list-recurring">
+                <li>
+                  <h4>{{$ctrl.recipient['year-to-date-amount'] | currency:'$'}}</h4>
+                  <span class="meta">{{$ctrl.currentDate | date:'mediumDate'}}</span>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div class="col-sm-3 col-md-3">
+            <div class="form-group mb0" ng-if="$ctrl.recipient['mostrecentdonation']">
+              <label translate>Most Recent:</label>
+              <ul class="list-unstyled list-recurring">
+                <li>
+                  <h4>{{$ctrl.recipient['mostrecentdonation']['historical-donation-line']['amount'] | currency:'$'}}</h4>
+                  <span class="meta">{{$ctrl.recipient['mostrecentdonation']['historical-donation-line']['transaction-date']['display-value'] | date:'mediumDate'}}</span>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div class="col-sm-4 col-md-4">
+            <div class="form-group mb0" ng-if="$ctrl.recipient['mostrecentdonationRecurringdonationelement']">
+              <label translate>Recurring Gifts:</label>
+              <ul class="list-unstyled list-recurring">
+                <li ng-repeat="recurring in $ctrl.recipient['mostrecentdonationRecurringdonationelement']">
```

Comment:
I haven't seen anything like that. Data I could find only had one item in "donation-lines", so I may not be displaying it properly, or it wasn't what I thought it was.